docs/how_keyboards_work.md

```diff
@@ -33,7 +33,11 @@ The firmware does not send actual letters or characters, but only scancodes.
 Thus, by modifying the firmware, you can only modify what scancode is sent over
 USB for a given key.
 
-## 3. What the Operating System Does
+## 3. What the Event Input/Kernel Does
+
+The *scancode* is mapped to a *keycode* dependant on the keyboard [60-keyboard.hwdb at Master](https://github.com/systemd/systemd/blob/master/hwdb/60-keyboard.hwdb). Without this mapping, the operating system will not receive a valid keycode and will be unable to do anything useful with that key press.
```

Comment:
A typo nitpick of sorts: `dependant` is typically used to describe "a person who relies on (is _dependent_) a person or organization for support," and only in British English; American English uses `dependent` for both the person and the adjective.

```suggestion
The *scancode* is mapped to a *keycode* dependent on the keyboard [60-keyboard.hwdb at Master](https://github.com/systemd/systemd/blob/master/hwdb/60-keyboard.hwdb). Without this mapping, the operating system will not receive a valid keycode and will be unable to do anything useful with that key press.
```

Reply:
Yes, a typo, I think it was late when I posted that.

Reply:
Ah. I thought maybe you might have been a British English speaker, in which case `dependant` wouldn't be wrong, but it would be inconsistent with the docs.